config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
At which point did you run `operator-sdk generate kustomize manifests -q`? This should be present in one of the previous commit for the current commit message to make sense.